varats-core/varats/report/gnu_time_report.py

```diff
@@ -272,14 +273,54 @@ def measurements_ctx_switches(self) -> tp.List[int]:
         """Context switches measurements of all aggregated reports."""
         return self._measurements_ctx_switches
 
+    @property
+    def max_resident_sizes(self) -> tp.List[int]:
+        return [report.max_res_size for report in self.reports()]
+
     @property
     def summary(self) -> str:
         return (
-            f"num_reports = {len(self.reports)}\n"
+            f"num_reports = {len(self.reports())}\n"
             "mean (std) of wall clock time = "
             f"{np.mean(self.measurements_wall_clock_time):.2f}"
             f" ({np.std(self.measurements_wall_clock_time):.2f})\n"
             "mean (std) of context switches = "
             f"{np.mean(self.measurements_ctx_switches):.2f}"
             f" ({np.std(self.measurements_ctx_switches):.2f})\n"
         )
+
+
+class WLTimeReportAggregate(
+    WorkloadSpecificReportAggregate[TimeReport],
+    shorthand=TimeReport.SHORTHAND + ReportAggregate.SHORTHAND,
+    file_type=ReportAggregate.FILE_TYPE
+):
+    """Context Manager for parsing multiple time reports stored inside a zip
+    file."""
```

Comment:
I suggest a better doc string. How about the following?
```suggestion
    """Context Manager for parsing multiple time reports stored inside a zip
    file and grouping them based on the workload they belong to."""
```